### PR TITLE
Hotfix/#8

### PR DESCRIPTION
**Pull request by FlavioCFOliveira**
Hello Dan,

Check if it makes sence to you add the follow instructions to the step 2

### CODE REVIEW

File: src/site/markdown/nexussetup.md.vm [hunk 1]
```diff
@@ -20,10 +20,20 @@ Nexus Setup
      to add the installer package repository. You can run the last step to install Nexus Repository Manager, but then
      you should immediately stop the Nexus daemon to allow you to customize the memory settings for the Pi. For example:
 
-        sudo apt-get install nexus-repository-manager
+        # Setup APT Repository and install
+        wget -qO - https://nx-staging.sonatype.com/repository/community-hosted/pki/deb-gpg/DEB-GPG-KEY-Sonatype.asc | sudo apt-key add -
```
Comment (bhamail) [inline src/site/markdown/nexussetup.md.vm:24]: Hm. I still think this duplicates the information that is linked to on line 19 above (specifically: [Apt Setup](https://github.com/sonatype-nexus-community/nexus-repository-installer#apt-setup)). Is there some other way we can emphasize that link? And/or maybe the steps at that link should be updated?

Comment (FlavioCFOliveira) [reply src/site/markdown/nexussetup.md.vm:24]: Yes, what is described on ln19 is enough but forces the user to navigate for a different page to setup the repository and come back to continue the instalation.

If you have on the same page the information for all the repositories, is more friendly to follow all the instalation steps. As for example here [Install MongoDB Community Edition on...](https://docs.mongodb.com/manual/tutorial/install-mongodb-on-ubuntu/)

My intention is not to duplicate niformation, is just a try to get a more straight forward "copy+paste" guide

As it is right now is good, and it works for me, but as a user it is more confortable to see all the steps in one place.

Thanks for your time and dedication

Comment (bhamail) [reply src/site/markdown/nexussetup.md.vm:24]: The more I look at your changes, the more I like them, even if they duplicate some of the original source. I think I'm gonna copy your approach of piping the key file directly into the key add command over to the original project.
Let's go ahead and merge this in, and we'll deal the added burden of keeping the package repository links updated.

Comment (FlavioCFOliveira) [reply src/site/markdown/nexussetup.md.vm:24]: Thankx

**Comment by bhamail** [comment]
Oh, I forgot to ask you to add yourself as a contributor! I'm happy to accept a PR if you'd like to add your name to the list of guilty parties.

**Comment by FlavioCFOliveira** [comment]
Sure it will be a pleasure to be a contributor, let's go for it

**Pull request merged**